client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -906,6 +910,14 @@ export default {
       userId: 'userId'
     }),
 
+    canNotBeNegativeReport () {
```

Comment:
I see this name is long!
maybe `isNotNegativeReport` better?

Reply:
Its long but still readable imho